src/v/cluster/cluster_utils.h

```diff
@@ -32,6 +32,12 @@ namespace detail {
 
 ss::sstring stm_snapshot_key(const ss::sstring&, const model::ntp& ntp);
 
+ss::future<> move_persistent_state(
```

Comment:
Can we make this method name to be more specific to its purpose ? It is only moving `persisted_stm` snapshots